lib/src/io/node.dart

```diff
@@ -27,20 +30,33 @@ class FileSystemException {
 }
 
 class Stderr {
-  final Writable _stderr;
+  final Writable? _stderr;
 
   Stderr(this._stderr);
 
-  void write(Object object) => _stderr.write(object.toString());
+  void write(Object object) {
+    if (_stderr == null) {
+      console.error(object.toString());
```

Comment:
Note that `stderr.write()` doesn't append a newline, while `console.error()` does. If we _do_ decide we need to fake stderr in the browser, we'll need to build up a buffer here and only emit it once we hit an actual newline.